### ISSUE TITLE
me/calendars does not list shared calendars added through web client

### ISSUE THREAD

**Issue by sgtpenguin** [microsoftgraph/msgraph-sdk-javascript#522]
# Bug Report
Requests to me/calendars do not list shared calendars that are added in the user's outlook web client.  If user adds calendar through the iOS app, the calendar shows up in the graph client request.

## Prerequisites

-   [yes] Can you reproduce the problem?
-   [yes] Are you running the latest version? (^2.2.1)
-   [yes] Are you reporting to the correct repository?
-   [yes] Did you perform a cursory search?

## Steps to Reproduce

1. Add shared calendar you are not the owner of in the outlook web client
2. Query "me/calendars" and see that the calendar doesn't show up
3. Add the calendar through the iOS app (haven't tested other platforms)
4. Query "me/calendars" and see that it shows up

**Expected behavior:** See all user's calendars regardless of platform

**Actual behavior:** User calendar data does not seem to be synchronized between web and iOS environments

## Additional Context
I am building a calendar widget for a large organization that shows a shared calendar.  A query to "me/calendars" for each user must reveal this shared calendar.  Asking them all to download iOS app and re-add the calendar is not a sufficient solution.

## Usage Information
Request ID - Value of the `requestId` field if you are receiving a Graph API error response

**Comment by nikithauc** [comment]
@sgtpenguin

> Requests to me/calendars do not list shared calendars that are added in the user's outlook web client. If user adds calendar through the iOS app, the calendar shows up in the graph client request.

 Was this working for you previously? Is the error new? If yes, when did you first notice the error?

**Comment by sgtpenguin** [comment]
No it was never working.  Not sure of the exact timeline when the error was noticed, I took over the project from someone else, and the bug existed at the time.  So at least a few months.

**Comment by nikithauc** [comment]
@sgtpenguin  I am still waiting on an update from the service team. The last update I received was that a similar issue was reported and being looked at first.

I will keep you posted.

**Comment by sebastienlevert** [comment]
Coming back to this thread. Can you validate that after a period of time (24h) you are getting these items? This endpoint seems to take time to propagate for shared calendars. Please validate and let us know. Thanks!

**Comment by sgtpenguin** [comment]
to the bot.  feedback coming :)

**Comment by sgtpenguin** [comment]
Sorry for the late response.  This integration has been removed from the project and I am no longer actively working on it.  

Due to outside circumstances and lack of access,  I never could definitively figure out if the endpoint propagated in 24 hours – I think so – it turned out my client was adding the wrong calendar at first.  The next issue we ran into was that we were only getting past events through the Calendar, but I wasn't able to investigate that long enough to know the problem for sure.

Thanks for your attention on this.
